client/src/pages/SignUp/SignUpForm/useStyles.ts

```diff
@@ -1,35 +1,57 @@
 import { makeStyles } from '@material-ui/core/styles';
+import { custom } from '../../../themes/custom';
 
 const useStyles = makeStyles((theme) => ({
   form: {
     width: '100%', // Fix IE 11 issue.
     marginTop: theme.spacing(1),
   },
   label: {
-    fontSize: 19,
-    color: 'rgb(0,0,0,0.4)',
+    fontSize: 11,
+    fontWeight: 600,
+    color: custom.colors.black,
     paddingLeft: '5px',
+    marginTop: '18px',
+    marginLeft: -theme.shape.borderRadius,
+    lineHeight: '0px',
+    textTransform: 'uppercase',
   },
   inputs: {
-    marginTop: '.8rem',
-    height: '2rem',
-    padding: '5px',
+    height: 43,
+    fontSize: 15,
+    padding: '4px 18px 4px 18px',
   },
   forgot: {
     paddingRight: 10,
-    color: '#3a8dff',
+    color: custom.colors.red,
   },
   submit: {
-    margin: theme.spacing(3, 2, 2),
     padding: 10,
-    width: 160,
-    height: 56,
+    width: 130,
+    height: 44,
     borderRadius: theme.shape.borderRadius,
-    marginTop: 49,
-    fontSize: 16,
-    backgroundColor: '#3a8dff',
+    margin: '40px 0px 45px 0px',
+    fontSize: 12,
     fontWeight: 'bold',
+    textTransform: 'uppercase',
+    boxShadow: 'none',
   },
+  loginBox: {
+    display: 'flex',
+    justifyContent: 'center',
+  },
+  isMember: {
+    fontSize: 13,
```

Comment:
Optional: You can consider defining font sizes for variants within MUI's theme and use those variants throughout the app 😄

Reply:
Good suggestions. Thanks.